Enhanced_Zeus_Modules.sqf

```diff
@@ -11036,6 +11036,147 @@ MAZ_EZM_fnc_initFunction = {
 
 	comment "Gameplay";
 
+		HYPER_EZM_fnc_handleCreateIntel = {
+			params ["_title","_description","_intelObject","_deleteOnPickup", "_target", "_holdDuration", "_visibility"];
+
+			comment "we set the default model as the documents for both redundancy and so we dont have to have a default case below";
+			private _intelObjModel = "Item_Laptop_Unfolded";
+			switch (_intelObject) do {
+				case "laptop": {_intelObjModel = "Item_Laptop_Unfolded"};
+				case "ruggedtablet": {_intelObjModel = "Land_Tablet_02_black_F"};
+				case "tablet": {_intelObjModel = "Land_Tablet_01_F"};
+				case "documents": {_intelObjModel = "LanD_File1_f"};
+				case "folder": {_intelObjModel = "Land_File_research_F"};
+			};
+			private _intelObj = _intelObjModel createVehicle _target;
+			_intelObj setDamage 1;
+			_intelObj setPosATL _target;
+
+			private _actionParams = [
+				_intelObj,
+				"<t color='#ff9b00'>Pick up Intel</t>", 
+				"a3\ui_f\data\igui\cfg\holdactions\holdaction_search_ca.paa", 
+				"a3\ui_f\data\igui\cfg\holdactions\holdaction_search_ca.paa", 
+				"_this distance _target < 3",
+				"_caller distance _target < 3",  
+				{},
+				{},
+				{
+					private _intel = _this select 0; 
+					private _args = _this select 3;
+					_args params ["_title","_description","_deleteOnPickup", "_visibility"];
+
+					[_intel,0] remoteExec ["removeAction",0,true];
```

Comment:
Remove the `true` JIP parameter.

Reply:
Also does this even remove the action? It's a hold action.

Reply:
Yeah it should 
![image](https://github.com/user-attachments/assets/264caa36-2675-4fb3-8ed7-33433444e373)

Reply:
Huh, didn't know that. Cool